src/acceleration/test/AccelerationIntraCommTest.cpp

```diff
@@ -67,6 +67,8 @@ BOOST_AUTO_TEST_CASE(testVIQNILSpp)
   mesh::PtrData displacements(new mesh::Data("dvalues", -1, 1));
   mesh::PtrData forces(new mesh::Data("fvalues", -1, 1));
 
+  bool exchangeSubsteps = false; // @todo set "true" as soon as acceleration scheme supports subcycling
```

Comment:
Would be good to have test for both then.

Reply:
True. Will take care of this in #1696 and duplicate the test correspondingly.